sync/views.py

```diff
@@ -33,25 +31,27 @@ def sync(request):
         # This _could_ cause timeouts, but might be OK?
         # Will depend on how many line items we get.
         for knack_value in knack_values:
-            log_debug('sent {} to knack'.format(json.dumps(knack_value, indent=4)))
+            logger.info('Knack Sending {}'.format(
+                json.dumps(knack_value, indent=4)))
             order_id = actblue_data['contribution']['orderNumber']
             entity_id_key = settings.ACTBLUE_TO_KNACK_MAPPING_ARRAY_ITEMS['lineitems#entityId']
             lineitem_entity_id = knack_value[entity_id_key]
 
-            return_status, result_string = knackload.load(json.dumps(knack_value), knack_object_id)
+            return_status, result_string = knackload.load(
+                json.dumps(knack_value), knack_object_id)
 
             if return_status != 200:
-
-                error_msg = 'Error: We failed to send order {}, lineitem {} to knack'
-                print(error_msg.format(order_id, lineitem_entity_id))
-                log_debug('Error: We failed to send {}'.format(json.dumps(knack_value, indent=4)))
-
+                logger.error('Knack Send Failed', extra={
+                    'payload': knack_value,
```

Comment:
this means PII will be in the logs upon error. Can we just log the same fields we log upon success here, which should be enough to get us back to the ActBlue data that caused the problem?

Reply:
makes sense

Reply:
should be good now